beamer/state_machine.py

```diff
@@ -63,6 +65,9 @@ def process_event(event: Event, context: Context) -> bool:
     elif isinstance(event, ClaimWithdrawn):
         return _handle_claim_withdrawn(event, context)
 
+    elif isinstance(event, (RequestResolved, FillHashInvalidated)):
+        return False
+
```

Comment:
Do we have documentation for these events/transitions?

Reply:
What do you mean? This PR just adds the events and queries them. No state machine code is touched, but we need to ignore them here because we run into a panic otherwise.

Reply:
> What do you mean? This PR just adds the events and queries them. No state machine code is touched, but we need to ignore them here because we run into a panic otherwise.

Right, but that just means that the state machine remains in the same state upon receiving those events.
What I'm trying to say is, it would be nice to have that fact reflected in our state machine diagram.

A related question is: when can those events be emitted i.e. for which states could it happen that those events arrive?

Reply:
> Right, but that just means that the state machine remains in the same state upon receiving those events.
> What I'm trying to say is, it would be nice to have that fact reflected in our state machine diagram.

I would put it differently, we ignore them just as before. And as this is part of a larger effort to get L1 resolution working I don't see any point in adjusting the docs in this PR.